packages/components/input-number/style/themes/default.less

```
@@ -0,0 +1,21 @@
+@import '../../../style/themes/default.less';
+@import '../../../input/style/themes/default.less';

```

```suggestion
@import '../../../form/style/themes/default.less';

```
下面 input 的变量都改成以 form 的